jekyll/_data/sidenav.yml

```diff
@@ -291,6 +291,8 @@ en:
           link: using-matrix-jobs
         - name: Use dynamic configuration
           link: using-dynamic-configuration
+        - name: Avoiding and debugging Java memory errors
```

Comment:
Should we update this to "Avoid and debug Java memory errors"?

Reply:
@devinmarieb Ah yes thank you for the reminder!